pkg/keyring/keyring_windows.go

```diff
@@ -116,6 +116,18 @@ func SetBearer(prefix, secret string) error {
 	return saveEncryptedFile(bearer, prefix, secret)
 }
 
+func DeleteBearer(prefix string) error {
```

Comment:
This wont work, the bearer token is not stored in the keyring, since the size is to big, on windows its stored as a encrypted dpapi file on disk, I think what you want todo is this
https://github.com/appgate/sdpctl/blob/31162b8dfd2fa8ecdd1b8de47e0ae919a3f4a73d/pkg/keyring/keyring_windows.go#L34-L42

Reply:
ops! I thought that was a Mac specific problem. Anyway, made a quick PR here #296